haproxy/content.md

```diff
@@ -21,7 +21,10 @@ Note: Many configuration examples propose to put `daemon` into the `global` sect
 	FROM haproxy:1.5
 	COPY haproxy.cfg /usr/local/etc/haproxy/haproxy.cfg
 
-Build and run:
+## Test the configuration file	
+	docker run -it --rm  --name my-running-haproxy -v /path/to/haproxy.cfg:/usr/local/etc/haproxy/haproxy.cfg:ro haproxy:1.5 haproxy -c -f /usr/local/etc/haproxy/haproxy.cfg
```

Comment:
Since this section is about building an image, I think it would be safer and more generally likely to work on an arbitrary user's box if we separated this out to use the built image instead of a bind-mount, ie:
```markdown
    FROM haproxy:1.5
    COPY haproxy.cfg /usr/local/etc/haproxy/haproxy.cfg

## Build

    docker build -t my-haproxy .

## Test the configuration file

    docker run --rm my-haproxy haproxy -c -f /usr/local/etc/haproxy/haproxy.cfg

## Run

    docker run -d --name my-running-haproxy my-haproxy
```

Would you like to update your PR, or would you rather I carry the PR from here?  Thanks for pointing out the enhancement regardless! :+1: